code/modules/reagents/reactions/other.dm

```diff
@@ -275,3 +225,97 @@
 	name = "laughter"
 	results = list(/datum/reagent/consumable/laughter = 5)
 	required_reagents = list(/datum/reagent/consumable/sugar = 1, /datum/reagent/consumable/drink/banana = 1)
+	
+	
+//Explosives and pyrotechnics
+
+
+/datum/chemical_reaction/emp_pulse
+	name = "EMP Pulse"
+	required_reagents = list(/datum/reagent/uranium = 1, /datum/reagent/iron = 1) // Yes, laugh, it's the best recipe I could think of that makes a little bit of sense
+
+/datum/chemical_reaction/emp_pulse/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	// 100 created volume = 4 heavy range & 7 light range. A few tiles smaller than traitor EMP grandes.
+	// 200 created volume = 8 heavy range & 14 light range. 4 tiles larger than traitor EMP grenades.
+	empulse(location, round(created_volume / 24), round(created_volume / 14), 1)
+	holder.clear_reagents()
```

Comment:
It's not if tivi feels like removing it, it's either:
It's useful, it stays there.
It's not doing anything, you remove

And it's not up to us to tell you which is which

Reply:
It's useful because it's a chemical reaction that causes an EMP, and is currently the only way to create an EMP in game.

It's not useful because we don't use EMP's for anything, and the reaction is difficult to obtain. 
Both are true. I was leaving it in because I didn't alter it (besides moving it to group it better), and it's something I'm not smart enough to redo easily if we want it for anything.

Basically, I have no strong feelings on the matter.

Reply:
"t's not useful because we don't use EMP's for anything"

Ok, remove

Reply:
k